### PR TITLE
[ID] Add translation for Abandoned Wasteland

### PR DESCRIPTION
**Pull request by wowcake**

### CODE REVIEW

File: wiki/Abandoned_wasteland/id.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - forum
+  - wastelanded
+---
+
+# Abandoned wasteland
+
+**Abandoned wasteland** adalah sebuah [forum-sampingan](https://osu.ppy.sh/community/forums)<!-- TODO --> yang bersifat pribadi digunakan untuk menyembunyikan forum-forum<!-- TODO: link --> dan forum topik<!-- TODO: link --> agar tidak terlihat oleh publik. Forum-sampingan ini hanya bisa diakses oleh para moderator<!-- TODO: link --> yang mempunyai kemampuan untuk memodifikasi forum. Perintah kepada para moderator mengenai hal tersebut adalah: `Taruh hal-hal yang tidak kita butuhkan kedalam sini.`.
```
Comment (mousesalt) [inline wiki/Abandoned_wasteland/id.md:10]:
```suggestion
**Abandoned wasteland** adalah sebuah [subforum](https://osu.ppy.sh/community/forums)<!-- TODO --> yang bersifat pribadi digunakan untuk menyembunyikan forum-forum<!-- TODO: link --> dan topik-topik forum<!-- TODO: link --> agar tidak terlihat publik. Subforum ini hanya bisa diakses oleh para moderator<!-- TODO: link --> yang mempunyai kuasa untuk memodifikasi forum. Instruksi kepada para moderator mengenai hal tersebut adalah: `Taruh hal-hal yang tidak kita butuhkan kedalam sini.`.
```

Comment (mousesalt) [reply wiki/Abandoned_wasteland/id.md]: use subforum instead, to get understandable

File: wiki/Abandoned_wasteland/id.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - forum
+  - wastelanded
+---
+
+# Abandoned wasteland
```
Comment (mousesalt) [inline wiki/Abandoned_wasteland/id.md:8]:
```suggestion
# Gurun yang terbengkalai
```

Comment (mousesalt) [reply wiki/Abandoned_wasteland/id.md:8]: do we still need to use english, i think it's hard to understand for this one

File: wiki/Abandoned_wasteland/id.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+**Abandoned wasteland** adalah sebuah [forum-sampingan](https://osu.ppy.sh/community/forums)<!-- TODO --> yang bersifat pribadi digunakan untuk menyembunyikan forum-forum<!-- TODO: link --> dan forum topik<!-- TODO: link --> agar tidak terlihat oleh publik. Forum-sampingan ini hanya bisa diakses oleh para moderator<!-- TODO: link --> yang mempunyai kemampuan untuk memodifikasi forum. Perintah kepada para moderator mengenai hal tersebut adalah: `Taruh hal-hal yang tidak kita butuhkan kedalam sini.`.
+
+Forum-forum dan topik-topik disebut sebagai "wastelanded" ketika mereka dipindahkan ke dalam forum ini.
```
Comment (mousesalt) [inline wiki/Abandoned_wasteland/id.md:12]:
```suggestion
Forum-forum dan topik-topik disebut sebagai "terbengkalai" ketika mereka dipindahkan ke dalam forum ini.
```

File: wiki/Abandoned_wasteland/id.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - forum
+  - wastelanded
+---
```
Comment (mousesalt) [inline wiki/Abandoned_wasteland/id.md:6]:
```suggestion
- abandoned
---
```

File: wiki/Abandoned_wasteland/id.md (hunk 3 above)
Comment (wowcake) [reply wiki/Abandoned_wasteland/id.md:12]: i think we should follow the original term

File: wiki/Abandoned_wasteland/id.md (hunk 2 above)
Comment (wowcake) [reply wiki/Abandoned_wasteland/id.md:8]: we really should not

**Review by mousesalt — DISMISSED** [review]
should be fine on my sight :+1:

**Comment by wowcake** [comment]
wrong branch, whoops

**Pull request closed without merging**